cdc/tests/integration_tests/_utils/start_tidb_cluster_impl

```diff
@@ -219,15 +221,6 @@ tidb-server \
 	--status=${UP_TIDB_STATUS} \
 	--log-file "$OUT_DIR/tidb.log" &
 
-randomGenSocketsConf
```

Comment:
Suggest to use `tiup playground` to start the cluster, which would be easier, as we don't need to maintain all the commands & ports here. We just concern ports of PD.

But in current version of `tiup playground`, the PD port of downstream cluster is random (as the default 2379 is occupied by upstream cluster), which caused it difficult to make automatic tests. 

The good news is that the feature of specifying PD port will be released in a near version (v1.11.0, see https://github.com/pingcap/tiup/pull/1931).

So I think we can consider change to `tiup playground` later after the new version is released.

Reply:
If use `tiup playground`, we can't add/remove cdc_server casually for next test case.